pkg/operator/ceph/controller/spec_test.go

```diff
@@ -457,7 +457,7 @@ func TestLogCollectorContainer(t *testing.T) {
 		additionalLogFile := "/tmp/ceph_test_log_500KB.log"
 		maxsize, _ := resource.ParseQuantity("500K")
 		c := cephv1.ClusterSpec{LogCollector: cephv1.LogCollectorSpec{Enabled: true, Periodicity: "1d", MaxLogSize: &maxsize}}
-		got := LogCollectorContainer(daemonId, ns, c, additionalLogFile)
+		got := LogCollectorContainer(daemonId, ns, c, nil, additionalLogFile)
```

Comment:
@arttor just a suggestion: maybe a test for new envVar params might be good to have https://github.com/rook/rook/pull/15605/files#diff-a9297a5049e576e37ad570d32715fb869f71bfd088b4a056370f74d41307f4cdR802 to test the changes with this PR.